webdav3/client.py

```diff
@@ -234,12 +234,13 @@ def valid(self):
         return True if self.webdav.valid() else False
 
     @wrap_connection_error
-    def list(self, remote_path=root, get_info=False):
+    def list(self, remote_path=root, get_info=False, recursive=True):
```

Comment:
Also we need to write tests for new functionality otherwise Quality gate will be failed